consensus/parlia/parlia.go

```diff
@@ -94,6 +94,7 @@ var (
 		common.HexToAddress(systemcontracts.GovernorContract):           true,
 		common.HexToAddress(systemcontracts.GovTokenContract):           true,
 		common.HexToAddress(systemcontracts.TimelockContract):           true,
+		common.HexToAddress(systemcontracts.TokenRecoverPortalContract): true,
```

Comment:
No need to init `TokenRecoverPortalContract` contract

Reply:
Done